lib/Image.php

```diff
@@ -452,6 +452,46 @@ public function src( $size = 'full' ) {
 		return $src;
 	}
 
+	/**
+	 * @param string $size a size known to WordPress (like "medium")
+	 * @api
+	 * @example
+	 * ```twig
+	 * <h1>{{ post.title }}</h1>
+	 * <img src="{{ post.thumbnail.src }}" srcset="{{ post.thumnbail.srcset }}" />
+	 * ```
+	 * ```html
+	 * <img src="http://example.org/wp-content/uploads/2018/10/pic.jpg" srcset="http://example.org/wp-content/uploads/2018/10/pic.jpg 1024w, http://example.org/wp-content/uploads/2018/10/pic-600x338.jpg 600w, http://example.org/wp-content/uploads/2018/10/pic-300x169.jpg 300w" />
+	 * ```
+	 *	@return bool|string
+	 */
+	public function srcset( $size = "full" ) {
+		$srcset = wp_get_attachment_image_srcset($this->ID, $size);
+		$srcset = apply_filters('timber/image/srcset', $srcset, $this->ID);
+		$srcset = apply_filters('timber_image_srcset', $srcset, $this->ID);
+		return $srcset;
+	}
+	
+	/**
+	 * @param string $size a size known to WordPress (like "medium")
+	 * @api
+	 * @example
+	 * ```twig
+	 * <h1>{{ post.title }}</h1>
+	 * <img src="{{ post.thumbnail.src }}" srcset="{{ post.thumnbail.srcset }}" sizes="{{ post.thumbnail.sizes }}" />
+	 * ```
+	 * ```html
+	 * <img src="http://example.org/wp-content/uploads/2018/10/pic.jpg" srcset="http://example.org/wp-content/uploads/2018/10/pic.jpg 1024w, http://example.org/wp-content/uploads/2018/10/pic-600x338.jpg 600w, http://example.org/wp-content/uploads/2018/10/pic-300x169.jpg 300w sizes="(max-width: 1024px) 100vw, 102" />
+	 * ```
+	 *	@return bool|string
+	 */
+	public function img_sizes( $size = "full" ) {
+		$img_sizes = wp_get_attachment_image_sizes($this->ID, $size);
+		$img_sizes = apply_filters('timber/image/img_sizes', $img_sizes, $this->ID);
```

Comment:
Also here, I’m not sure if we need this filter at all, because `wp_get_attachment_image_sizes()` calls `wp_calculate_image_sizes()` in the last line, which in turn has a `wp_calculate_image_sizes` filter in the last line. So the `timber/image/img_sizes` filter would be called immediately after `wp_calculate_image_sizes`. What would be the benefit of this?